.github/actions/compile-commit/action.yml

```diff
@@ -38,7 +38,7 @@ runs:
         export MAVEN="./mvnw --offline"
         # maven.wagon.rto is in millis, defaults to 30m
         MAVEN_INSTALL_OPTS="-Xmx3G -XX:+ExitOnOutOfMemoryError -Dmaven.wagon.rto=60000"
-        MAVEN_COMPILE_COMMITS="-B --strict-checksums --quiet -T C1 -DskipTests -Dmaven.source.skip=true -Dair.check.skip-all=true -Dmaven.javadoc.skip=true --no-snapshot-updates --no-transfer-progress -pl !:trino-server-rpm"
+        MAVEN_COMPILE_COMMITS="-B --strict-checksums --quiet -DskipTests -Dmaven.source.skip=true -Dair.check.skip-all=true -Dmaven.javadoc.skip=true --no-snapshot-updates --no-transfer-progress -pl !:trino-server-rpm"
```

Comment:
Why is `-T C1` gone?

Reply:
That's interesting...

So, how are the plans to switch to `mvnd` going? ;)

Reply:
Scratch that. We've just been using it wrong ;)

Reply:
damn, I need to update all my scripts 😭

Reply:
FYI @findepi (for https://gist.github.com/findepi/04c96f0f60dcc95329f569bb0c44a0cd)